app/models/ability.rb

```diff
@@ -137,6 +138,8 @@ def signed_in_with_organizer_role(user)
     can :manage, Sponsor, conference_id: conf_ids_for_organizer
     can :manage, SponsorshipLevel, conference_id: conf_ids_for_organizer
     can :manage, Ticket, conference_id: conf_ids_for_organizer
+    can :index, Comment, commentable_type: 'Event',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.